content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,117 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you’ve just created the reference entity `Brand`, you’ve defined it’s records' attributes and now it’s time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
```

Comment:
I don't know if the use of contractions is appropriate for the documentation, in any way if you want to use them, it could be interesting to know if you want to say its or it's (contraction of it is). This sentence is not pretty clear. I would say instead: " Let’s say that you have just created the reference entity `Brand`, you’ve defined its attributes records and now it is time to add new ones. For example: Fermob, Kartell, Muuto, Fatboy…"